src/components/PreviewComment/index.js

```diff
@@ -131,7 +131,7 @@ export const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: COLORS.blackgrey,
+    color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color '#828282' should be replaced with a constant for better maintainability.
```diff
- color: '#828282',
+ color: colors.grey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->